### PR TITLE
21w41a monster XP fields

### PR DESCRIPTION
**Pull request by apple502j**
Not entirely sure about the meaning, but I don't think there are that much constants with value 0, 3, 5, 10, 20 and 50.

### CODE REVIEW

**Comment by liach** [comment]
I recall monsters have xp bonus for extra armors. Also seems baby monsters drop 11 xp, which appears not in this list.

File: mappings/net/minecraft/entity/mob/Monster.mapping [hunk 1]
```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_1569 net/minecraft/entity/mob/Monster
+	FIELD field_30464 ZERO_XP I
+	FIELD field_30465 SMALL_MONSTER_XP I
+	FIELD field_30466 NORMAL_MONSTER_XP I
+	FIELD field_30467 STRONG_MONSTER_XP I
+	FIELD field_30468 STRONGER_MONSTER_XP I
+	FIELD field_30469 WITHER_XP I
```
Comment (enbrain) [inline mappings/net/minecraft/entity/mob/Monster.mapping]: Maybe document that these values are used for `MobEntity#experiencePoints`. A link to this class might be useful as well.

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/entity/mob/Monster.mapping [hunk 2]
```diff
@@ -1 +1,23 @@
...
+	FIELD field_30467 STRONG_MONSTER_XP I
+	COMMENT @apiNote This is used for monsters such as
+	COMMENT {@linkplain BlazeEntity blazes} and
+	COMMENT {@linkplain GuardianEntity guardians}.
+	FIELD field_30468 STRONGER_MONSTER_XP I
+	COMMENT @apiNote This is used for {@linkplain PiglinBruteEntity piglin brutes}
+	COMMENT and {@linkplain RavagerEntity ravagers}.
+	FIELD field_30469 WITHER_XP I
+	COMMENT @apiNote This is used for
+	COMMENT {@linkplain net.minecraft.entity.boss.WitherEntity withers}.
```
Comment (enbrain) [inline mappings/net/minecraft/entity/mob/Monster.mapping]:
```suggestion
	FIELD field_30464 ZERO_XP I
		COMMENT @apiNote This is used for mobs that do not drop experience.
	FIELD field_30465 SMALL_MONSTER_XP I
		COMMENT @apiNote This is used for monsters such as
		COMMENT {@linkplain EndermiteEntity endermites} and
		COMMENT {@linkplain VexEntity vexes}.
	FIELD field_30466 NORMAL_MONSTER_XP I
		COMMENT @apiNote This is used for most of the monsters, such as
		COMMENT {@linkplain ZombieEntity zombies} and
		COMMENT {@linkplain SkeletonEntity skeletons}.
	FIELD field_30467 STRONG_MONSTER_XP I
		COMMENT @apiNote This is used for monsters such as
		COMMENT {@linkplain BlazeEntity blazes} and
		COMMENT {@linkplain GuardianEntity guardians}.
	FIELD field_30468 STRONGER_MONSTER_XP I
		COMMENT @apiNote This is used for {@linkplain PiglinBruteEntity piglin brutes}
		COMMENT and {@linkplain RavagerEntity ravagers}.
	FIELD field_30469 WITHER_XP I
		COMMENT @apiNote This is used for
		COMMENT {@linkplain net.minecraft.entity.boss.WitherEntity withers}.
```

**Review by enbrain — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/mob/Monster.mapping [hunk 3]
```diff
@@ -1 +1,23 @@
 CLASS net/minecraft/class_1569 net/minecraft/entity/mob/Monster
+	COMMENT This interface contains constants to be used for
```
Comment (liach) [inline mappings/net/minecraft/entity/mob/Monster.mapping]: I recall the interface is mainly a marker for hostile mobs (monsters) than just a constant holder.

Comment (enbrain) [reply mappings/net/minecraft/entity/mob/Monster.mapping]: Yes. That's why I didn't suggest a rename to `MonsterConstants`. Additional documentation wouldn't hurt though.

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/entity/mob/Monster.mapping (hunk 3 above)
Comment (liach) [reply mappings/net/minecraft/entity/mob/Monster.mapping]:
```suggestion
	COMMENT This interface marks that an entity class belongs to monsters, including
	COMMENT hostile mobs like zombies. It also contains constants to be used for
```

File: mappings/net/minecraft/entity/mob/Monster.mapping [hunk 4]
```diff
@@ -1 +1,23 @@
...
+	COMMENT setting {@link MobEntity#experiencePoints}.
```
Comment (liach) [inline mappings/net/minecraft/entity/mob/Monster.mapping:4]:
```suggestion
	COMMENT setting {@link MobEntity#experiencePoints}.
	COMMENT
	COMMENT @see net.minecraft.entity.SpawnGroup#MONSTER
```

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**